### PR TITLE
Added an accessibility button and that saves states between classes

### PR DESCRIPTION
**Pull request by sourenrex**

### CODE REVIEW

File: app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java [hunk 1]
```diff
@@ -48,6 +57,12 @@ public void updateRoutineView(Workout workout) {
         routineLayout.addView(button);
 
     }
+    public void back() {
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java:60]: Forgot @Override

File: app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java [hunk 2]
```diff
@@ -24,12 +27,16 @@ protected void onCreate(Bundle savedInstanceState) {
         // Retrieve UI
         this.routineLayout = findViewById(R.id.RoutineLayout);
         Button addWorkoutBtn = findViewById(R.id.AddWorkoutBtn);
+        Button back = findViewById(R.id.backButton2);
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java:30]: would be nicer to make this backBtn (following the convention of other variables of type Button)

File: app/src/main/java/com/example/fitappa/View/ViewRoutinesActivity.java [hunk 3]
```diff
@@ -25,9 +26,10 @@ protected void onCreate(Bundle savedInstanceState) {
         // Load the elements
         routinesLayout = findViewById(R.id.routinesLayout);
         Button createRoutineBtn = findViewById(R.id.CreateRoutineBtn);
+        Button back = findViewById(R.id.backButton);
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewRoutinesActivity.java:29]: would be nicer to make this backBtn (following the convention of other variables of type Button)

File: app/src/main/java/com/example/fitappa/View/ViewRoutinesActivity.java [hunk 4]
```diff
@@ -51,6 +54,12 @@ public void updateRoutinesView(Routine routine) {
         routinesLayout.addView(button);
 
     }
+    @Override
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewRoutinesActivity.java:57]: add javadoc

File: app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java (hunk 1 above)
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java:60]: add javadoc

**Comment by nxabdullah** [comment]
Looks good, just take a look at the Java doc comment and you're good to merge

File: app/src/main/java/com/example/fitappa/View/ViewWorkoutActivity.java [hunk 5]
```diff
@@ -40,6 +48,12 @@ private void openAddExercise() {
         addExerciseIntent.putExtra("workoutObj", this.workout);
         startActivityForResult(addExerciseIntent, 1);
     }
+    public void back() {
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/ViewWorkoutActivity.java:51]: add @Override and javadoc

**Pull request merged**